src/OrleansServiceFabricUtils/FabricServiceSiloResolver.cs

```diff
@@ -132,14 +133,6 @@ private void OnPartitionChange(long handlerId, FabricPartitionResolutionChange a
                     if (!existing.IsSamePartitionAs(updated.Partition)) continue;
                     found = true;
 
-                    if (updated.Partition.IsOlderThan(existing))
-                    {
-                        this.log.Info($"Update for partition {updated} is superseded by existing version.");
-
-                        // Do not update the partition if the exiting one has a newer version than the update.
-                        break;
```

Comment:
I'm not an expert on service fabric. But would removing this post the risk of older partition overwrites newer partition? Yes the older partition would mostly be corrected by newer partition eventually. But this brought unnecessary handling on partitionChange, which can be avoided, right?  or is this pre-mature optimization?